play-media-native/screens/CreateAthlete/General.tsx

```diff
@@ -0,0 +1,54 @@
+import { useState } from "react";
+import { View, TextInput } from "react-native";
+import { Text } from "react-native-paper";
+import { DatePicker } from "../../components/DatePicker/DatePicker";
+import { SportPicker } from "../../features/SportPicker/SportPicker";
+import { getDate } from "../../helpers/dateHelper";
+import { Sport } from "../../interfaces/sport";
+import { theme } from "../../theme/theme";
+import { athleteStyles } from "./styles";
+
+type GeneralProps = {
+  sports: Sport[];
+};
+
+export const General = ({ sports }: GeneralProps) => {
+  const [name, handleChangeName] = useState("");
+  const [nationality, handleChangeNationality] = useState("");
+  const [birthDate, handleChangeBirthDate] = useState(new Date());
+  const [showBirthDatePicker, setShowBirthDatePicker] = useState(false);
+
+  return (
+    <View style={{ marginTop: theme.spacing.lg }}>
+      <Text style={athleteStyles.label}>Athlete name</Text>
+      <TextInput
```

Comment:
Use InputText. We can reuse the same styles everywhere and change them in one place if input styles require changing.  Same goes for input title. Or a required star. Neli did some work on that to align styles with designs.